Update swift-tools-version to allow the recently updated iOS version

Current Package.swift leads to the build error:

```
/Package.swift:32:13: error: 'v13' is unavailable
...
note: 'v13' was introduced in PackageDescription 5.1
```

--- a/Package.swift
+++ b/Package.swift
@@ -1,4 +1,4 @@
-// swift-tools-version:5.0
+// swift-tools-version:5.3
 //
 //  Package.swift
 //  NextLevelSessionExporter (http://nextlevel.engineering/)

